osu.Android/OsuGameAndroid.cs

```diff
@@ -55,6 +55,12 @@ public override Version AssemblyVersion
             }
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            LoadComponentAsync(new GameplayScreenRotationLocker(), Add);
```

Comment:
Is there a reason this needs to be loaded async here?  Can you not just cache it like any other component?

Reply:
There isn't a real reason for this, apart from trying to do things consistently with desktop